
syntax = "proto3";

package logits;

service LogitsService {
    rpc GetLogits(LogitsRequest) returns (LogitsResponse);
}

message LogitsRequest {
    string input_ids = 1;
    string attention_mask = 2;
    string model_config = 3;
}

message LogitsResponse {
    string logits = 1;
    bool success = 2;
    string error_message = 3;
}
